tests/v2/validation/charts/vsphere_cpi_cis_test.go

```diff
@@ -0,0 +1,195 @@
+package charts
+
+import (
+	"fmt"
+	"os"
+	"testing"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/rancher/rancher/tests/framework/clients/rancher"
+	management "github.com/rancher/rancher/tests/framework/clients/rancher/generated/management/v3"
+	"github.com/rancher/rancher/tests/framework/extensions/charts"
+	"github.com/rancher/rancher/tests/framework/extensions/clusters"
+	"github.com/rancher/rancher/tests/framework/pkg/config"
+	"github.com/rancher/rancher/tests/framework/pkg/session"
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+)
+
+const (
+	// Project that charts are installed in
+	deplymentNamespace    = "default"
+	vsphereCpiProjectName = "system"
+	CpiConfigFileKey      = "cpi"
+	CsiConfigFileKey      = "csi"
+)
+
+// chartInstallOptions is a private struct that has cpi and csi charts install options
+type vsphereChartInstallOptions struct {
+	cpi *charts.InstallOptions
+	csi *charts.InstallOptions
+}
+
+type CpiTestSuite struct {
+	suite.Suite
+	client                     *rancher.Client
+	session                    *session.Session
+	project                    *management.Project
+	vsphereChartInstallOptions *vsphereChartInstallOptions
+	cpiConfig                  *charts.CpiConfig
+	csiConfig                  *charts.CsiConfig
+}
+
+func (c *CpiTestSuite) TearDownSuite() {
+	c.session.Cleanup()
+}
+
+func (c *CpiTestSuite) SetupSuite() {
+	testSession := session.NewSession()
+	c.session = testSession
+
+	client, err := rancher.NewClient("", testSession)
+	require.NoError(c.T(), err)
+
+	c.client = client
+
+	cpiConfig := new(charts.CpiConfig)
+	csiConfig := new(charts.CsiConfig)
+	config.LoadConfig(CpiConfigFileKey, cpiConfig)
```

Comment:
Can you start this on a new line? This code is a bit cluttered together, making it hard to read.